Soccar [Client]/src/nl/soccar/ui/GameCanvas.java

```diff
@@ -20,14 +24,29 @@ public GameCanvas() {
     
     public abstract void stop();
 
+    /**
+     * Method that adds a drawable to the list of drawables.
+     * 
+     * @param drawable Drawable item that needs to be added to the list of drawables.
+     */
     public final void addDrawable(Drawable drawable) {
         drawables.add(drawable);
     }
 
+    /**
+     * Method that removes drawable of the list of drawables.
+     * 
+     * @param drawable Drawable item that needs to be removed from the list.

```

Comment:
@param drawable Drawable item that needs to be removed from the **drawable** list.